config/sara/sara.yml

```diff
@@ -66,6 +66,11 @@ show_demo_warning_background: <%= ENV["SHOW_DEMO_WARNING_BACKGROUND"] || false %
 # If this message is set, it will be displayed at the top of the page.
 banner_message: <%= ENV["BANNER_MESSAGE"] || '' %>
 
+# If this URL is set, it will be included in messages sent to moniitorees
+# For jurisdictions that use a custom intro SMS message, the privacy policy info will automatically be appended to the custom message
+# if this link is not already included in the message
```

Comment:
Again, if you agree with my other comment, this line should be removed
```suggestion
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1247/commits/8fac5befa1faba9061b8ef057506327d991f57f2